pkg/reconciler/apis/apibinding/apibinding_controller.go

```diff
@@ -211,6 +197,23 @@ func NewController(
 		UpdateFunc: func(_, obj interface{}) { c.enqueueAPIExport(obj, logger, "") },
 		DeleteFunc: func(obj interface{}) { c.enqueueAPIExport(obj, logger, "") },
 	})
+
+	// APIExport-related indexers
+	indexers.AddIfNotPresentOrDie(apiExportInformer.Informer().GetIndexer(), cache.Indexers{
+		indexers.ByLogicalClusterPathAndName: indexers.IndexByLogicalClusterPathAndName,
+		indexAPIExportsByAPIResourceSchema:   indexAPIExportsByAPIResourceSchemasFunc,
+	})
+	indexers.AddIfNotPresentOrDie(globalAPIExportInformer.Informer().GetIndexer(), cache.Indexers{
+		indexers.ByLogicalClusterPathAndName: indexers.IndexByLogicalClusterPathAndName,
+		indexAPIExportsByAPIResourceSchema:   indexAPIExportsByAPIResourceSchemasFunc,
+	})
```

Comment:
I used to put all indexers first, then all event handlers. I assumed this is a convention we follow.